### PR TITLE
docs: Release notes for 0.13.4.

### PR DESCRIPTION
**Pull request by justin-determined-ai**
## Test Plan

Built and clicked around.

## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234

### CODE REVIEW

File: docs/release-notes.txt [hunk 1]
```diff
@@ -8,6 +8,53 @@
...
+Version 0.13.4
+==============
+
+**Release Date:** September 16, 2020
+
+**Improvements**
+
+- Warn users when setting ``DistributeConfig`` on an Estimator. Configuring an Estimator via ``tf.distribute.Strategy`` can conflict with how Determined performs distributed training. With this change, Determined will attempt to catch this problem and surface an error message in the experiment logs. An Estimator can still be configured with an empty ``DistributeConfig`` without issue.
+
+- Support configuring default task images, Docker pull policy, and Docker registry credentials via the :ref:`master-configuration` and the :ref:`helm-config`. Note that individual task configurations will continue to overwrite these defaults.
```
Comment (aaron276h) [inline docs/release-notes.txt:20]: question: do the :ref: links get rendered when we publish release notes?

Comment (neilconway) [reply docs/release-notes.txt:20]: Yes.

Comment (justin-determined-ai) [reply docs/release-notes.txt:20]: Looks like this:

Screen Shot 2020-09-16 at 15 10 10

File: docs/release-notes.txt [hunk 2]
```diff
@@ -8,6 +8,53 @@
...
+
+- WebUI: Move TensorBoard sources from a table column into a separate modal.
+
+- WebUI: Reduce the time it takes to load active TensorBoards and notebooks.
+
+**Bug Fixes**
+
+- Improve handling of certain corner cases where distributed training jobs could hang indefinitely.
+
+- Fix an issue where detecting GPU availability in TensorFlow code would cause EstimatorTrial models to OOM.
```
Comment (determined-dsw) [inline docs/release-notes.txt:46]: EstimatorTrial should be double-backticked.

File: docs/release-notes.txt [hunk 3]
```diff
@@ -8,6 +8,53 @@
...
 **************
 
+Version 0.13.4
+==============
+
+**Release Date:** September 16, 2020
+
+**Improvements**
+
+- Warn users when setting ``DistributeConfig`` on an Estimator. Configuring an Estimator via ``tf.distribute.Strategy`` can conflict with how Determined performs distributed training. With this change, Determined will attempt to catch this problem and surface an error message in the experiment logs. An Estimator can still be configured with an empty ``DistributeConfig`` without issue.
```
Comment (determined-dsw) [inline docs/release-notes.txt]: Estimator should be double-backticked.

File: docs/release-notes.txt [hunk 4]
```diff
@@ -8,6 +8,53 @@
...
+
+- Support configuring default task images, Docker pull policy, and Docker registry credentials via the :ref:`master-configuration` and the :ref:`helm-config`. Note that individual task configurations will continue to overwrite these defaults.
+
+- Add connection checks for dynamic agents. A dynamically provisioned agent will be terminated if it is not actively connected to the master for at least five minutes.
+
+- Remove support for ``dataflow_to_tf_dataset`` in EstimatorTrial. Dataflows should be wrapped using ``wrap_dataset(shard=false)`` instead.
```
Comment (determined-dsw) [inline docs/release-notes.txt]: EstimatorTrial should be double-backticked.

File: docs/release-notes.txt (hunk 2 above)
Comment (neilconway) [reply docs/release-notes.txt]: fixed.

File: docs/release-notes.txt (hunk 4 above)
Comment (neilconway) [reply docs/release-notes.txt]: fixed

**Comment by determined-dsw** [comment]
Should we have categories for all rows, not just "WebUI"?

**Review by neilconway — APPROVED** [review]
Pushed some fixes, otherwise lgtm.

**Comment by justin-determined-ai** [comment]
@determined-dsw Yeah, let's have a discussion about this async :)

**Pull request merged**